### PR TITLE
Change "a SSL" to "an SSL"

### PR DESCRIPTION
**Pull request by andrewkdinh**
@mattcaswell 
- "a SSL" -> "an SSL"
- "A SSL" -> "An SSL"

- [x] documentation is added or updated

### CODE REVIEW

**Review by mattcaswell — DISMISSED** [review]

**Review by tom-cosgrove-arm — DISMISSED** [review]
LGTM

**Comment by t8m** [comment]
Oh, this is targeted against the feature/quic-server branch. This is wrong. @andrewkdinh Could you please target this against the master branch?

**Comment by tom-cosgrove-arm** [comment]
@andrewkdinh Yes please (although I am not @t8m!). The quic branch can merge in those fixes from master. Then the final remaining changes on the quic branch can be done there

**Comment by andrewkdinh** [comment]
@tom-cosgrove-arm @mattcaswell can you please re-review after I changed baes branch to `master`? I will do a separate PR for changes specific to feature/quic-server

File: demos/bio/sconnect.c [hunk 1]
```diff
@@ -59,7 +59,7 @@ int main(int argc, char *argv[])
     if (!SSL_CTX_load_verify_locations(ssl_ctx, CAfile, NULL))
         goto err;
 
-    /* Lets make a SSL structure */
+    /* Lets make an SSL structure */
```
Comment (tom-cosgrove-arm) [inline demos/bio/sconnect.c:62]: (minor; I didn't spot this before) this should be `Let's` - let's fix this since we're changing this line anyway

**Review by shahsb — APPROVED** [review]
LGTM.

Thanks for making these changes! Every improvement counts.

File: demos/bio/sconnect.c (hunk 1 above)
Comment (andrewkdinh) [reply demos/bio/sconnect.c]: Updated :+1:

**Review by mattcaswell — APPROVED** [review]

**Comment by mattcaswell** [comment]
@tom-cosgrove-arm - please reconfirm

**Comment by mattcaswell** [comment]
Side note: It would be quite nice in some future PR to add a check for this error in the find-doc-nits script.

**Review by tom-cosgrove-arm — APPROVED** [review]
Reconfirming

**Pull request closed without merging**